pdns/dnsdistdist/docs/reference/config.rst

```diff
@@ -806,7 +806,9 @@ A server object returned by :func:`getServer` can be manipulated with these func
 
   .. method:: Server:isUp() -> bool
 
-    Returns the up status of the server
+    Returns the up status of the server.
+    Result is based on the administrative status of the server (as set by either :meth:`Server:setDown` or :meth:`Server:setUp`).
+    If no administrative status is set, result is based on :attr:`Server.upStatus`
```

Comment:
How do you set "no admin status"? A ref to `setAuto` would help I think.

Reply:
Agreed! The current situation is a bit messy, though, see https://github.com/PowerDNS/pdns/issues/13836#issuecomment-1967099059 So, @gjherbiet, if you don't want to dig into it that's fine by me, I can merge this PR and do a follow-up one :)

Reply:
Or i can just add "If no administrative status is set (see :meth:`Server:setAuto`)" so the reference is there, and let you further elaborate later.

Reply:
Sounds good to me, thanks!

Reply:
Done in 2ffb3645e24b714ab661037341676a2478ac38f7.